diff --git a/recipes-browser/chromium/chromium-gn.inc b/recipes-browser/chromium/chromium-gn.inc
index 7cc2e65..e496512 100644
--- a/recipes-browser/chromium/chromium-gn.inc
+++ b/recipes-browser/chromium/chromium-gn.inc
@@ -78,7 +78,7 @@ DEPENDS += " \
     pkgconfig-native \
     ${@bb.utils.contains('DISTRO_FEATURES', 'pulseaudio', 'pulseaudio', '', d)} \
     qemu-native \
-    virtual/libgl \
+     \
 "
 
 # The wrapper script we use from upstream requires bash.
@@ -102,7 +102,7 @@ PACKAGECONFIG ??= "use-egl"
 
 # this makes sure the dependencies for the EGL mode are present; otherwise, the configure scripts
 # automatically and silently fall back to GLX
-PACKAGECONFIG[use-egl] = ",,virtual/egl virtual/libgles2"
+PACKAGECONFIG[use-egl] = ",,virtual/egl es2"
 
 # Empty PACKAGECONFIG options listed here to avoid warnings.
 # The .bb file should use these to conditionally add patches
