diff --git a/CMakeLists.txt b/CMakeLists.txt
index 918d0da..a687f4c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,10 +76,10 @@ find_package(Wayland REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(Spnav QUIET)
 
-if (${PC_GSTREAMER_VERSION} VERSION_LESS "1.6.0")
-    message(WARNING "Using old GStreamer!")
-    add_definitions(-DGST_PAD_PROBE_HANDLED=GST_PAD_PROBE_DROP)
-endif()
+#if (${PC_GSTREAMER_VERSION} VERSION_LESS "1.6.0")
+#    message(WARNING "Using old GStreamer!")
+#    add_definitions(-DGST_PAD_PROBE_HANDLED=GST_PAD_PROBE_DROP)
+#endif()
 
 #find_package(libsv REQUIRED CONFIG)
 
@@ -157,6 +157,7 @@ set(
     m
     rt
     mediactl-v4l2
+    dl
 )
 
 if (SPNAV_FOUND)
